nixos/modules/security/pam.nix

```diff
@@ -419,6 +431,8 @@ let
 
           # Password management.
           password sufficient pam_unix.so nullok sha512
+          ${optionalString cfg.enableHomed
+          "-password [success=1 default=ignore] ${pkgs.systemd}/lib/security/pam_systemd_home.so"}
```

Comment:
I'm not sure about the right parameters and order here - this still seems to be somewaht WIP:

https://github.com/systemd/systemd/issues/16906

Reply:
I took this from an arch vm. Not exactly sure how it works, but it seems to work under arch